tests/graphql/test-add-field.graphql

```diff
@@ -0,0 +1,42 @@
+#> server-sdl
+schema @server @upstream(baseURL: "http://jsonplacheholder.typicode.com") {
+  query: Query
+}
+
+type A {
+  b: B
+}
+
+type B {
+  c: String
+}
+
+type Foo {
+  a: A
+}
+
+type Query @field(name: "b", path: ["foo", "a", "b"]) {
+  foo: Foo @http(path: "/foo")
+}
```

Comment:
Need some execution tests. And failing tests for paths that have a resolver. Feel free to copy paste all the `@inline` tests. That will allow us to delete `@inline` confidently.